tests/CMakeLists.txt

```diff
@@ -150,3 +150,26 @@ if (PY_MYPY)
         COMMAND env "${PYTHON_EXECUTABLE}" -m mypy "${CMAKE_SOURCE_DIR}/utils/import_srg_spreadsheet.py"
      )
 endif()
+
+if (PY_OPENPYXL)
+    add_test(
+        NAME "srg-export-rhel9"
+        COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/utils/create_srg_export.py" --root "${CMAKE_SOURCE_DIR}" --json "${CMAKE_SOURCE_DIR}/build/rule_dirs.json" --control "${CMAKE_SOURCE_DIR}/controls/srg_gpos.yml" --product rhel9 --out-format xlsx --output "${CMAKE_BINARY_DIR}/cac_stig_output.xlsx" --build-config-yaml "${CMAKE_BINARY_DIR}/build_config.yml"
+    )
+    set_tests_properties("srg-export-rhel9" PROPERTIES LABELS quick)
+    set_tests_properties("srg-export-rhel9" PROPERTIES DEPENDS "test-rule-dir-json")
```

Comment:
I don't like the DEPENDS on other tests, because I can't run the test standalone using the `-R` option. I was looking into fixtures https://cmake.org/cmake/help/latest/prop_test/FIXTURES_REQUIRED.html but they are new in CMake 3.7. So I would propose to create the rule_dir.json as a part of this test.

Reply:
I will need to do some research but my initial reaction to that is that we might not be be able to use fixtures as RHEL 7 has CMake 2.7.